src/Symfony/Component/Serializer/Mapping/AttributeMetadata.php

```diff
@@ -110,24 +114,66 @@ public function getMaxDepth(): ?int
         return $this->maxDepth;
     }
 
-    public function setSerializedName(?string $serializedName): void
+    public function setSerializedName(?string $serializedName /* , array $groups = [] */): void
```

Comment:
If `$serializedName` is null, we must unset the data in `serializedNames`

Reply:
That's what I originally wanted to do, but my laziness won😁